common/src/main/java/com/wynntils/models/players/PlayerModel.java

```diff
@@ -190,6 +200,21 @@ private void loadUser(UUID uuid, String userName) {
                 });
     }
 
+    public CompletableFuture<com.wynntils.models.players.type.PlayerInfo> getPlayer(String username) {
+        CompletableFuture<com.wynntils.models.players.type.PlayerInfo> future = new CompletableFuture<>();
```

Comment:
Seems like IDEA had a stroke when refactoring this...

Reply:
Or.. MC already has a `PlayerInfo` class. Rename to `WynnPlayerInfo` perhaps?

Reply:
Yeah already has one and both are already used in the model